pkg/config/config.go

```diff
@@ -320,21 +325,10 @@ type UnsupportedConfigOverrides struct {
 }
 
 // ApplyEmbeddedUnsupportedOverrides applies the custom configuration to the cluster config.
-func ApplyEmbeddedUnsupportedOverrides(config *v1beta1.Cluster, embconfig []byte) error {
-	if embconfig == nil {
+func ApplyEmbeddedUnsupportedOverrides(config *v1beta1.Cluster, embconfig string) error {
+	if embconfig == "" {
 		return nil
 	}
-	var overrides UnsupportedConfigOverrides
```

Comment:
I think we should drop this struct, useless stuff.

Reply:
done